runtime/src/main.rs

```diff
@@ -119,7 +122,15 @@ extern "C" fn nmi_handler(trap_record: &TrapRecord) {
         ext_info.at(4).write(|_| err_interrupt_status);
     }
 
-    handle_fatal_error(caliptra_drivers::CaliptraError::RUNTIME_GLOBAL_NMI.into());
+    let mut error = CaliptraError::RUNTIME_GLOBAL_NMI;
+
+    let wdt_status = soc_ifc.regs().cptra_wdt_status().read();
+    if wdt_status.t1_timeout() || wdt_status.t2_timeout() {
+        cprintln!("WDT Expired");
+        error = CaliptraError::RUNTIME_GLOBAL_WDT_EXPIRED;
+    }
```

Comment:
tiny nit, could do this to be a little more idiomatic

```suggestion
    let wdt_status = soc_ifc.regs().cptra_wdt_status().read();
    let error = if wdt_status.t1_timeout() || wdt_status.t2_timeout() {
        cprintln!("WDT Expired");
        CaliptraError::RUNTIME_GLOBAL_WDT_EXPIRED
    } else {
        CaliptraError::RUNTIME_GLOBAL_NMI
    }
```